pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -32,7 +33,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/wait"
 )
 
-// managerImpl implements NodeStabilityManager
+// managerImpl implements Manager
```

Comment:
yes, the type assertion is already below the struct def.  i was just fixing an inaccuracy in the in comment.